health/micro-ui/web/core/install-deps.sh

```
@@ -0,0 +1,26 @@
+        exit 1
+    fi
+    cp "$1" "$2" || { echo "Error: Failed to copy $1 to $2"; exit 1; }
+}
+
+copy_file "core/App.js" "src"
+copy_file "core/package.json" "package.json"
+copy_file "core/webpack.config.js" "webpack.config.js"
+copy_file "core/inter-package.json" "$INTERNALS/package.json"
+cp $INTERNALS/example/src/UICustomizations.js src/Customisations
+
+echo "UI :: core " && echo "Branch: $(git branch --show-current)" && echo "$(git log -1 --pretty=%B)" && echo "installing packages" 
```

Shellcheck (0.10.0)

[style] Useless echo? Instead of 'echo $(cmd)', just use 'cmd'.

(SC2005)